default/stringtables/en.txt

```diff
@@ -5077,34 +5077,34 @@ BEGINNER_HINT_04
 Systems that are connected by thick lines are [[encyclopedia SUPPLY_TITLE]] line linked and they share [[encyclopedia INDUSTRY_TITLE]] Production Points (PP).
 
 BEGINNER_HINT_05
-All system share [[encyclopedia RESEARCH_TITLE]] Points (RP) that they generate even if they are not connected by supply lines. If you have a remote system that is unconnected, setting the [[encyclopedia FOCUS_TITLE]] to research might be a more efficient use.
+All system share [[encyclopedia RESEARCH_TITLE]] Points (RP) that they generate even if they are not connected by supply lines.  When a system is not connected, setting the [[encyclopedia FOCUS_TITLE]] to research might be more efficient than other settings.
 
 BEGINNER_HINT_06
-Ships will refuel when they are in a system that is connected by thick or by thin lines, as long as the thin lines extend back to thick supply line connected systems.
+Ships will refuel when they are in a system that is connected to a [[encyclopedia SUPPLY_TITLE]] line.  Narrow supply lines, which are displayed with a thin line between systems, will also refuel ships if they are connected to a wider supply line.
```

Comment:
Normal starlanes are also displayed by a thin line between systems. Probably should mention the coloring somewhere.